gulpfile.js

```diff
@@ -140,10 +152,18 @@ gulp.task('prod', function (cb) {
     runSequence('build-prod', 'server', cb);
 });
 
-gulp.task('ci', ['build-ci'], function () {
-    return gulp.src(['dist/**', '!dist/hud.zip'])
+gulp.task('archive-dev', ['build-dev'], function () {
+    return gulp.src(['.dist/dev/**'])
+        .pipe(zip('hud.zip'))
```

Comment:
Could you not just name it `hud-dev.zip` here rather than rename it later?  (Or are you trying to end up with both a `hud.zip` and `hud-dev.zip` in the `dev` folder?